selfdrive/controls/controlsd.py

```diff
@@ -307,6 +307,8 @@ def data_send(plan, path_plan, CS, CI, CP, VM, state, events, actuators, v_cruis
     CC.hudControl.leadVisible = plan.hasLead
     CC.hudControl.rightLaneVisible = bool(path_plan.pathPlan.rProb > 0.5)
     CC.hudControl.leftLaneVisible = bool(path_plan.pathPlan.lProb > 0.5)
+    CC.hudControl.rightLaneDepart = bool(path_plan.pathPlan.rPoly[3] > -1.11 and not CS.rightBlinker) and CS.vEgo > 12.5 # Speed needs to be above 12.5m/s for LDA and only if blinker if off
```

Comment:
Where does the magic value `-1.11`  and `1.05` below come from?

Reply:
They are test values from center of car to model line

Reply:
Taking into account that the car is 2m wide and the detected lines don't seem to go underneath 1m